pyface/ui/qt4/workbench/split_tab_widget.py

```diff
@@ -324,7 +321,7 @@ def _focus_changed(self, old, new):
             else:
                 nw = ntw.widget(ntidx)
 
-            self.emit(QtCore.SIGNAL('hasFocus'), nw)
```

Comment:
Probably because that's what PyQt expected back in the day.  Or it could have been a bug.